polly/include/polly/ScopDetection.h

```diff
@@ -345,6 +345,20 @@ class ScopDetection {
   /// @return True if region is profitable to optimize, false otherwise.
   bool isProfitableRegion(DetectionContext &Context) const;
 
+  /// Check if an expanded region is profitable to optimize.
+  ///
+  /// An expanded region may include basic blocks with memory accesses that
+  /// are not used in loops of the expanded region. These memory accesses add
```

Comment:
The actual test looks for any memory access, not just accesses not used in the loop.

Reply:
I have updated the checks to look for memory accesses that are not used in loops.